### PR TITLE
Small cleanups

### PR DESCRIPTION
**Pull request by vincent-legoll**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,30 +4,28 @@
...
+| `openio_gridinit_conf_confd` | `/etc/gridinit.d` | Path to the service's folder (by namespace) |
+| `openio_gridinit_rundir` | `/run/gridinit` | Path to the tmpfs subfolder |
+| `openio_gridinit_limits` | `dict` | Defines the max open files and limits for coredump |
 | `openio_gridinit_services` | `[]` | Defines services to configure |
 
-
 ## Dependencies
 
-- You have to use this role after the role `ansible-role-openio-repository` (like [this](https://github.com/open-io/ansible-role-openio-gridinit/blob/docker-tests/test.yml#L7))
+- You have to use this role after the role `ansible-role-repo-openio-sds` (like [this](https://github.com/open-io/ansible-role-openio-gridinit/blob/docker-tests/test.yml#L7))
```
Comment (cdelgehier) [inline README.md]: Wrong name
https://github.com/open-io/ansible-role-openio-repository

File: README.md [hunk 2]
```diff
@@ -66,12 +64,12 @@ An Ansible role for gridinit. Specifically, the responsibilities of this role ar
 
 
 ```ini
-[servers]
+[all]
```
Comment (cdelgehier) [inline README.md]: Not really a best practice

**Review by cdelgehier — CHANGES_REQUESTED** [review]
Rollback `ansible-role-openio-repository` and it s good

File: README.md (hunk 1 above)
Comment (vincent-legoll) [reply README.md]: Yep you're right

File: README.md (hunk 2 above)
Comment (vincent-legoll) [reply README.md]: It was so that it is the same as the group the playbook is applied to...

**Comment by vincent-legoll** [comment]
ping

**Pull request merged**